storage/object/api-cli/multipart-uploads.mdx

```diff
@@ -310,6 +310,6 @@ $ aws s3api list-multipart-uploads --bucket my-bucket
 ```
 
 <Navigation title="See Also">
-  <PreviousButton to="/storage/object/api-cli/lifecycle-rules-api/">Managing the lifecycle of objects</PreviousButton>
+  <PreviousButton to="/storage/object/api-cli/managing-lifecycle-cliv2">Managing an Object Storage Lifecycle using CLI (v2)</PreviousButton>
```

Comment:
the Next button points to /storage/object/api-cli/setting-cors-rules/" Setting CORS rules, and that page exists but doesn't have a link on the side (navigation) menu